handlers/dimensions.go

```diff
@@ -544,15 +590,22 @@ func (f *Filter) AddList(w http.ResponseWriter, req *http.Request) {
 		options = append(options, k)
 	}
 
-	if err := f.FilterClient.AddDimensionValues(req.Context(), filterID, name, options); err != nil {
+	if err := f.FilterClient.AddDimensionValues(req.Context(), userAccessToken, serviceAuthToken, collectionID, filterID, name, options); err != nil {
 		log.InfoCtx(req.Context(), err.Error(), nil)
 	}
 
 	http.Redirect(w, req, redirectURL, 302)
 }
 
-func (f *Filter) getDimensionValues(ctx context.Context, filterID, name string) (values []string, labelIDMap map[string]string, err error) {
-	fj, err := f.FilterClient.GetJobState(ctx, filterID)
+func (f *Filter) getDimensionValues(ctx context.Context, userAccessToken, serviceAuthToken, filterID, name string) (values []string, labelIDMap map[string]string, err error) {
+
+	collectionID := getCollectionIDFromContext(ctx)
+	if err != nil{
```

Comment:
Use `!headers.IsNotFound(err)` 

Reply:
Error check was a ghost check and now removed.